src/components/alert/alert.scss

```diff
@@ -149,3 +153,10 @@
     transform: scale(scale.$effect-shrink);
   }
 }
+
+.c-alert__error {
```

Comment:
This appears to be an element class (`__error`) but it should be a modifier class (`--error`).